SEP-0002.md

```diff
@@ -3,203 +3,288 @@
 | SEP           | 2                                                       |
 |---------------|---------------------------------------------------------|
 | title         | SunPy Organization Definition                           |
-| author(s)     | [Steven Christe](https://orcid.org/0000-0001-6127-795X) |
-| contact email | <steven.d.christe@nasa.gov>                               |
-| date-creation | 2014-02-16                                              |
+| author(s)     | [Stuart Mumford](https://orcid.org/0000-0003-4217-4642) |
+| contact email | <stuart@cadair.com>                                     |
+| date-creation | 2024-04-09 (Original version: 2014-02-16)               |
 | type          | process                                                 |
-| discussion    | unavailable                                             |
-| status        | accepted                                                |
-
-## Abstract
-
-The SunPy organization currently lacks a formal organization
-structure. This proposal defines the SunPy organization which manages
-the code and related properties (e.g. docs, website, etc.)
-
-## Detailed Description
-
-The organization of SunPy is modeled on that usually adopted by
-non-profit organizations (e.g.
-[501c](http://en.wikipedia.org/wiki/501(c)_organization). This
-organization structure is tried and tested and has the added advantage
-that it should provide a smooth transition to incorporating a SunPy
-foundation if it becomes necessary in the future (e.g. for funding
-purposes). The primary role of the organization is to *facilitate and
-promote the use and development of community-led, free and
-open-source solar data-analysis software based on the scientific
-Python environment*. This includes but is not limited to the following
-tasks
-
-* developing the SunPy software library
-* manage and protect the SunPy
-brand and identity
-* promote SunPy to the community
-* managing SunPy assets, i.e. github account, domain name etc.
-
-The organization consists of an executive director, an advisory board, and the
-developer community. The goal of the proposed structure is to provide and
-promote community input into the project.
+| discussion    | <https://github.com/sunpy/sunpy-SEP/pull/62>            |
+| status        | in review                                               |
+
+## Rationale
+
+This SEP describes the second major iteration of the governance for the SunPy Project.
+The objectives of the rewrite were to adapt the governance for the changes the project has seen since the first iteration of this document was written in 2014.
```

Comment:
How's this?
```suggestion
The objectives of this update are to optimize the governance based on our experiences running the project since 2014 when the original governing SEP was written. That original governance structure was modeled after a non-profit company structure, assumed a very committed board, and intertwined project finances with board responsibilities.  The project has since matured and responsibilities for project management should be fully given over to the community while a smaller board with a reduced mandate provides high-level oversight.
```